lib/tasks/scripts/telemedicine_reports.rb

```diff
@@ -3,32 +3,80 @@
 require "csv"
 
 class TelemedicineReports
-  attr_reader :mixpanel_csv_path, :period_start, :period_end, :mixpanel_data, :report_array
+  attr_reader :period_start, :period_end, :mixpanel_data, :report_array
 
-  def initialize(mixpanel_csv_path, period_start, period_end)
-    @mixpanel_csv_path = mixpanel_csv_path
+  def initialize(period_start, period_end)
     @period_start = period_start
     @period_end = period_end
     @mixpanel_data = {hydrated: [], formatted: []}
     @report_array = []
+    @filename = "telemedicine_report_#{@period_start.strftime("%d_%b")}_to_#{@period_end.strftime("%d_%b")}.csv"
+    @query = <<~QUERY
+      function main() {
+        return Events({
+          from_date: "#{period_start.strftime("%Y-%m-%d")}",
+          to_date: "#{period_end.strftime("%Y-%m-%d")}"
+        }).filter(function(event) {
+          return event.name == "UserInteraction" && event.properties.name == "Patient Summary:Contact Doctor Clicked"
+        }).groupBy(
+          ["properties.$user_id",
+          function(event) {
+              return (new Date(event.time)).toDateString();
+            }],
+          mixpanel.reducer.count()
+        );
+      }
+    QUERY
+  end
+
+  def fetch_mixpanel_data
+    url = URI.parse("https://mixpanel.com/api/2.0/jql")
+
+    response = HTTP.basic_auth(user: ENV["MIXPANEL_API_SECRET"], pass: nil)
```

Comment:
Shall we `ENV.fetch` here so it fails loud and clear if not configured? Eg. If we turn it on in BD accidentally.